.dependabot/config.yml

```diff
@@ -4,6 +4,31 @@ update_configs:
   - package_manager: 'javascript'
     directory: '/'
     update_schedule: 'weekly'
+    automerged_updates:
+      - match:
+          dependency_name: 'chai*'
+          update_type: 'semver:minor'
```

Comment:
Is the `update_type` specified here literal or more like a semver ceiling? For example will this only auto-merge on a literal minor version bump, or will it work on both patch _and_ minor?

Reply:
My reading of the docs is its the second one: `update_type: 'semver:minor'` should auto-merge patch and minor updates (like the inverse of `@dependabot ignore this minor`).